apps/passport-client/components/modals/RequireAddPasswordModal.tsx

```diff
@@ -69,13 +71,13 @@ export function RequireAddPasswordModal(): JSX.Element {
       end-to-end-encrypted Zupass. To upgrade, please choose a password. Make
       sure to remember it, otherwise you will lose access to all your PCDs.
       <Spacer h={24} />
-      <BigInput value={self.email} disabled={true} />
+      <BigInput value={self?.email} disabled={true} />
```

Comment:
```suggestion
      <BigInput value={self?.email ?? ""} disabled={true} />
```

Reply:
Annoying that the type doesn't enforce this.